core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/util/ClassLoaderHelper.java

```diff
@@ -25,22 +25,20 @@ private ClassLoaderHelper() {
 
     private static Class<?> loadClassViaClasses(String fqcn, Class<?>[] classes) {
         if (classes != null) {
```

Comment:
null check is redundant as its now done in classLoader method

Reply:
I'd still prefer leaving it here coz we might add other caller methods w/o the null check in the future.